src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,127 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int CAPACITY_OF_ELEMENT_DATA = 10;
+    private T[] elementData;
+    private int size;
+
+    public ArrayList() {
+        elementData = (T[]) new Object[CAPACITY_OF_ELEMENT_DATA];
+        size = 0;
+    }
```

Comment:
```suggestion
    public ArrayList() {
        elementData = (T[]) new Object[CAPACITY_OF_ELEMENT_DATA];
    }
```

we don't need this, it will be initialized by 0 by default